src/goto-programs/goto_inline.cpp

```diff
@@ -826,8 +826,17 @@ void goto_partial_inline(
         it=goto_functions.function_map.begin();
         it!=goto_functions.function_map.end();
         it++)
-      if(it->second.body_available())
+    {
```

Comment:
This isn't in the PR. Please click through to the real diff to see what's actually in it.